ui/src/lib/types.ts

```
@@ -0,0 +1,65 @@
+}
+
+export interface Extras {
+    [key: string]: any;
+}
+
+export interface WebSocketCommand {
+    command: string;
+    payload?: string;
+}
+
+export type StartFunction<T> = (set: (value: T) => void) => (() => void) | void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)